pages/addComputer.ts

```diff
@@ -0,0 +1,73 @@
+// node ./pages/addComputer.ts
+
+import { expect, Locator, Page } from '@playwright/test';
+
+import { randomCompAndTime } from '../tests/controller.js'
+import { addComp_URL } from '../tests/config';
+import { LowerAlphaChar } from '@faker-js/faker/modules/string/index.js';
+import { exec } from 'child_process';
+
+//console.log(randomCompandTime);
+//console.log(addComp_URL);
+
+export class addCompPage {
+  readonly page: Page;
+  readonly addCompHeader: Locator;
+  readonly compNameField: Locator;
+  readonly introducedDate: Locator;
+  readonly discounted: Locator;
+  readonly company: Locator;
+  readonly createButton: Locator;
+  readonly cancelButton: Locator;
+  
+  constructor(page: Page) {
+    this.page = page;
+    this.addCompHeader = page.getByRole('heading', { name: 'Add a computer' });
+    this.compNameField = page.getByLabel('Computer name');
+    this.introducedDate = page.getByLabel('Introduced');
+    this.discounted = page.getByLabel('Discontinued');
+    this.company = page.getByRole('combobox', { name: 'Company' });
+    this.createButton = page.getByRole('button', { name: 'Create this computer' });
+    this.cancelButton = page.getByRole('link', { name: 'Cancel' });
+  }
+
+  async goto() {
+    await this.page.goto(addComp_URL);
+    await expect(this.addCompHeader).toBeVisible();
+  }
+  
+  async fillCompName() {
```

Comment:
Лучше дать возможность передать имя компании через аргумент функции:

```suggestion
  async fillCompName(text) {
     await this.compNameField.fill(text);
  }
```